### PR TITLE
Implement #155: Cleanup task output

### PR DESCRIPTION
**Pull request by M8is**
Implement #155: Cleanup task output

Since a picture is worth a thousand words, here's an example of the current output:

screenshot from 2018-03-28 13-35-22

### CODE REVIEW

File: mubench.pipeline/utils/logging_colorization.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+
+# inspired by: https://stackoverflow.com/a/1336640
+def __add_coloring_to_emit_ansi(fn):
+    RED = '\x1b[31m'
+    YELLOW = '\x1b[33m'
+    GREEN = '\x1b[32m'
+    NO_COLOR = '\x1b[0m'
+
+    # add methods we need to the class
```
Comment (salsolatragus) [inline mubench.pipeline/utils/logging_colorization.py]: Not sure what this comment is supposed to tell me.

File: mubench.pipeline/utils/logging_colorization.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    def new(*args):
+        level = args[0].levelno
+        if level == logging.CRITICAL:
+            color = RED  # red
```
Comment (salsolatragus) [inline mubench.pipeline/utils/logging_colorization.py]: Remove comments with color names.

File: mubench.pipeline/benchmark.py [hunk 3]
```diff
@@ -46,10 +46,18 @@ def run(self) -> None:
 
 logger = logging.getLogger()
 logger.setLevel(logging.DEBUG)
+
 handler = logging.StreamHandler()
-handler.setFormatter(IndentFormatter("%(indent)s%(message)s"))
 handler.setLevel(logging.INFO)
+logging_colorization.colorize(handler)
```
[the remark this answers is not in the document]
Comment (M8is) [reply mubench.pipeline/benchmark.py]: We should probably discuss this in person. It seems more convoluted than I thought.

[the remark this answers is not in the document]
Comment (salsolatragus) [reply mubench.pipeline/benchmark.py]: Ok, let's do this, then.

**Pull request merged**